hasher-matcher-actioner/hmalib/common/writebacker_models.py

```diff
@@ -176,8 +231,52 @@ class ThreatExchangeTruePositiveWritebacker(ThreatExchangeWritebacker):
     """
 
     def _writeback_impl(self, writeback_signal: BankedSignal) -> str:
-        # TODO Implement
-        return f"MOCKED: Wrote back TruePositive for indicator {writeback_signal.banked_content_id}"
+        indicator_id = writeback_signal.banked_content_id
+        privacy_group_id = writeback_signal.bank_id
+        descriptors = self.te_api.get_threat_descriptors_from_indicator(indicator_id)
+
+        # If we already have a descriptor we can copy it and upload make sure to never expire it
+        my_descriptor = None
+        for descriptor in descriptors:
+            if descriptor["owner"] == self.te_api.app_id:
+                my_descriptor = descriptor
+
+        privacy_members = str(privacy_group_id)
+
+        postParams = {
+            "privacy_type": "HAS_PRIVACY_GROUP",
+            "expire_time": 0,
+            "privacy_members": privacy_members,
+            "review_status": "REVIEWED_MANUALLY",
+            "status": "MALICIOUS",
+        }
+
+        if my_descriptor:
+            members = my_descriptor.get("privacy_members", "")
+            if members and privacy_members not in members:
```

Comment:
what if `privacy_members` (i.e. `privacy_group_id` see nit about confusing name above) is in members already? I'm not sure but I think it is possible that depending on what copy does for existing descriptor that this will remove it from the existing other privacy groups...